unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ChatController/ChatControllerContextMenuUtility.cs

```diff
@@ -0,0 +1,177 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using DCL.Interface;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+namespace DCL.Chat
+{
+    public partial class ChatController
```

Comment:
Love the usage of partial class! I was about to ask for this until I saw it!

Reply:
yes, it is an improvement, but im not really confortable with the approach though.. i think it is better to have a custom inspector and run this code in the editor assembly